panopticum/models.py

```diff
@@ -472,10 +424,8 @@ class ComponentVersionModel(models.Model):
     qa_upgrade_tests_signoff = panopticum.fields.SigneeField(related_name='signed_upgrade_tests')
 
     # meta
-
-    meta_update_by = models.ForeignKey(User, on_delete=models.PROTECT, blank=True, null=True, related_name='updater_of')
-    meta_update_date = models.DateTimeField(db_index=True)
-    meta_deleted = models.BooleanField(default=False)
+    update_date = models.DateTimeField(db_index=True, auto_now=True)
```

Comment:
we either need to rely on history (updated by, updated date) or keep this meta_ fields. So either both meta_update* disappear or both exist. No?

Reply:
For history and audit we will use 3td party library [django-simple-history](https://django-simple-history.readthedocs.io/en/latest/index.html). That library allow as to get information about when, who and what did update. And that fields does not needed. May be we will deprecate ```update_date``` too.

We already collect history info about ComponentVersion changes and can get that info by REST api.
https://github.com/perfguru87/panopticum/blob/master/panopticum_django/urls.py#L31